addons/website_slides/tests/common.py

```diff
@@ -16,6 +16,11 @@ def setUpClass(cls):
             groups='base.group_user,website_slides.group_website_slides_officer'
         )
 
+        cls.user_officer_other = mail_new_test_user(
```

Comment:
Since it's a bit specific to your test, maybe you can create it locally for now?